SimG4Core/Application/src/ParametrisedEMPhysics.cc

```diff
@@ -302,6 +304,7 @@ void ParametrisedEMPhysics::ConstructProcess() {
     ModifyTransportation(G4Positron::Positron(), nt, th1, th2);
     ModifyTransportation(G4Proton::Proton(), nt, th1, th2);
   }
+  G4cout << "ParametrisedEMPhysics::ConstructProcess() is done" << G4endl;
```

Comment:
@civanch why inside CMSSW code do we use Geant4 messages?

Reply:
@fabiocos , this is not 100% CMS code - it is an implementation of Geant4 derived class. G4cout is redirected to LogVerbatim, so it is equivalent. However, this may be changed.

Problem of this PR - sim history changed, this means that it is needed to understand why. I had a hope not break histories. The problem, which is fixed here is not obvious, more study required.